dvc/command/imp.py

```diff
@@ -65,4 +66,15 @@ def add_parser(subparsers, parent_parser):
         help="Specify name of the DVC-file this command will generate.",
         metavar="<filename>",
     )
+    import_parser.add_argument(
+        "-s",
+        "--store",
+        action="store_true",
+        default=False,
+        help=(
+            "Store imported files in this repository's DVC remote, and "
+            "retrieve files from the remote on 'dvc pull' "
+            "instead of reimporting."
```

Comment:
How about

"Enable pushing the imported data to remote storage, and pulling it from there (instead of reimporting)."

? Thanks